### DISCUSSION TITLE
Could you provide additional filelds for organizations?

### DISCUSSION THREAD

**Discussion by DevITSharpPRO** [SaintAngeLs/distributed_minispace#347] [Q&A] [answered]
I recently saw the  organization's document you are presenting. Could you please add the country, addresses and possible email phone as the contact data for specific organization?

**Comment by SaintAngeLs** [comment, marked as answer]
@DevITSharpPRO , thank you for your request. I agree this updated generally is needed. We will implement it in slightly next iteration.
